CanvasPlusPlayground/CourseAnnouncementManager.swift

```diff
@@ -26,14 +26,6 @@ import Foundation
                 self.announcements = cached.sorted(by: { 
                     ($0.createdAt ?? Date()) > ($1.createdAt ?? Date())
                 })
-            },
```

Comment:
Do we want to remove this?

Reply:
I mean the `onNewBatch` closure that you removed. I thought that was only for testing purposes.

Reply:
It's trivial, safer to omit for now.

Reply:
OK, but I'm still using it for people